tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_component.ts

```diff
@@ -13,28 +13,52 @@ See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
 
-import {ChangeDetectionStrategy, Component, Input} from '@angular/core';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  EventEmitter,
+  Input,
+  Output,
+} from '@angular/core';
 import {AxisDirection} from './linked_time_fob_controller_component';
 
 @Component({
   selector: 'linked-time-fob',
-  template: `<span [style.transform]="getCenteringTransform()">{{
-    step | number
-  }}</span>`,
+  templateUrl: 'linked_time_fob_component.ng.html',
   styleUrls: ['linked_time_fob_component.css'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class LinkedTimeFobComponent {
-  @Input()
-  step!: number;
-
+  @Input() step!: number;
   @Input() axisDirection!: AxisDirection;
 
+  @Output() stepChange = new EventEmitter<number>();
+
+  isTyping = false;
+
   getCenteringTransform() {
+    console.log('centering');
```

Comment:
There are a couple places in this file where you call console.log().

Reply:
Opps, some last minute checks that I forgot to remove. They are now removed.